src/app/landing/LandingPage.css

```diff
@@ -176,6 +176,18 @@
   justify-content: center;
 }
 
+.LandingPage .Feedback .SocialIconsList {
+  margin-top: -5%;
+  margin-left: -3px;
```

Comment:
Ugh, negative margins... always a sign of trouble.

Is it because a `ul` is used?
And if so, does it have to be?

Reply:
> Ugh, negative margins... always a sign of trouble.
> 
> Is it because a `ul` is used?
> And if so, does it have to be?

I restructured the icons so they're simple `<a>` tags now. This helped with the negative left margin, however the `<h2>` tags on the page all have quite a large bottom margin that I needed to reduce.